packages/fxa-auth-server/lib/routes/recovery-phone.ts

```diff
@@ -227,8 +227,6 @@ class RecoveryPhoneHandler {
       throw AppError.invalidToken();
     }
 
-    await this.customs.check(request, email, 'recoveryPhoneAvailable');
```

Comment:
If we want to add rate limiting for some authenticated endpoints, we could use [checkAuthenticated](https://github.com/mozilla/fxa/blob/088f2d7fd035aba6523091d82bde5fd22a38afd6/packages/fxa-customs-server/lib/server.js#L435) api. This will only check the uid and not ip or email address record. 

The rate limit is more generous though [(100 checks every 15mins)](https://github.com/mozilla/fxa/blob/088f2d7fd035aba6523091d82bde5fd22a38afd6/packages/fxa-customs-server/lib/config/config.js#L149) per action.